src/libs/actions/OnyxUpdateManager/utils/DeferredUpdates.ts

```diff
@@ -0,0 +1,68 @@
+import Onyx from 'react-native-onyx';
+import type {DeferredUpdatesDictionary} from '@libs/actions/OnyxUpdateManager/types';
+import * as SequentialQueue from '@libs/Network/SequentialQueue';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {OnyxUpdatesFromServer} from '@src/types/onyx';
+import createProxyForObject from '@src/utils/createProxyForObject';
+// eslint-disable-next-line import/no-cycle
+import * as OnyxUpdateManagerUtils from '.';
+import GetMissingOnyxUpdatesPromiseProxy from './GetMissingOnyxUpdatesPromise';
+
+let deferredUpdates: DeferredUpdatesDictionary = {};
+
+/**
+ * Manually processes and applies the updates from the deferred updates queue. (used e.g. for push notifications)
+ */
+function processDeferredUpdates() {
```

Comment:
```suggestion
function process() {
```

shouldn't need to repeat the name in these functions